parsers/yamlparser.go

```diff
@@ -229,7 +229,7 @@ type Project struct {
 	Namespace        string               `yaml:"namespace"`
 	Credential       string               `yaml:"credential"`
 	ApiHost          string               `yaml:"apiHost"`
-	ApigwAccessToken string               `yaml:"apigwAccessToken"`
+	ApigwAccessToken string               `yaml:"apigwAccessToken"` // TODO: support apigwTenantId? deprecate?
```

Comment:
Users potentially could use the wrong CRN# in yml file, and if that happens, the value would've been overwritten by the values in wskprops file. The api would be deployed in a namespace unexpected by the users.

Reply:
That is the intent, that is the values provided in the manifest/deployment files are per-deployment overrides... if they fail that is OK.  Also, it seems that if you have more than 1 IAM namespace, there needs to be a way to toggle between them more dynamically.  Looking at my account, i have 2 "default" IAM namespaces now... there is no UI element (or command line) to allow me to say which is truly my "Default" now; therefore, without a way to set this manually, the one auto-set in .wskprops seems to pick one of the 2 arbitrarily...  These options allow user to decide and explicitly provide the namespace they intended (via this mechanism, while treating the .wskprops value as the default)